_src/pattern-library/sass/components/_buttons.scss

```diff
@@ -57,6 +57,10 @@
 .btn-base {
     padding: spacing-vertical(x-small) spacing-horizontal(base);
     font-size: font-size(base);
+
+    svg {
```

Comment:
I think we need to assume some level of Sass expertise with this, no? I decided to use the size mixin so that this could be customized. A font-size will work here, as will a set pixel/rem/em value. I'm defaulting it to match the size of the accompanying text. This okay with you?